medusa/providers/torrent/html/yggtorrent.py

```diff
@@ -0,0 +1,174 @@
+        :param mode: The current mode used to search, e.g. RSS
+
+        :return: A list of items found
+        """
+        items = []
+
+        with BS4Parser(data, 'html5lib') as html:
+            torrent_table = html.find(class_='table table-striped')
+            torrent_rows = torrent_table('tr') if torrent_table else []
+
+            # Continue only if at least one Release is found
+            if len(torrent_rows) < 2:
+                log.debug('Data returned from provider does not contain any torrents')
+                return items
+
+            # Skip column headers
+            for result in torrent_rows[1:]:
+                cells = result('td')
+                if len(cells) < 5:
+                    continue
+                try:
+                    title = cells[0].find('a', class_='torrent-name').get_text(strip=True)
+                    download_url = urljoin(self.url, cells[0].find('a', target='_blank')['href'])
+                    if not (title and download_url):
+                        continue
+
+                    seeders = try_int(cells[4].get_text(strip=True))
+                    leechers = try_int(cells[5].get_text(strip=True))
+
+                    torrent_size = cells[3].get_text()
+                    size = convert_size(torrent_size, sep='') or -1
+
+                    # Filter unseeded torrent
+                    if seeders < min(self.minseed, 1):
+                        if mode != 'RSS':
+                            log.debug("Discarding torrent because it doesn't meet the"
+                                      " minimum seeders: {0}. Seeders: {1}",
+                                      title, seeders)
+                        continue
+
+                    item = {
+                        'title': title,
+                        'link': download_url,
+                        'size': size,
+                        'seeders': seeders,
+                        'leechers': leechers,
+                        'pubdate': None,
+                    }
+                    if mode != 'RSS':
+                        log.debug('Found result: {0} with {1} seeders and {2} leechers',
+                                  title, seeders, leechers)
+
+                    items.append(item)
+                except (AttributeError, TypeError, KeyError, ValueError, IndexError):
+                    log.exception('Failed parsing provider.')
+
+        return items
+
+    def login(self):
+        """Login method used for logging in before doing search and torrent downloads."""
+        login_params = {
+            'id': self.username,
+            'pass': self.password,
+            'submit': ''
+        }
+
+        self.session.post(self.urls['login'], data=login_params)
+        response = self.session.get(self.url)
+        if not response:
+            log.warning('Unable to connect to provider')
+            return False
+
+        if 'Connexion' in response.text:
```

Comment:
The thing is, I decided to go with Connexion, as that's a broader catch of exceptions. As I don't know if it will only respond to bad username/passwords entered? Maybe an account is disabled or banned, and a different message is shown.